registry.nuon

```diff
@@ -26,21 +26,166 @@
 
 {
     git: [
-        [name version url revision path];
         [
-            nu-git-manager
-            0.5.0
-            https://github.com/amtoine/nu-git-manager.git
-            0.5.0
+            name,
+            version,
+            url,
+            revision,
+            path
+        ];
+        [
+            nu_plugin_audio_hook,
+            "0.1.12",
+            "https://github.com/FMotalleb/nu_plugin_audio_hook",
+            "0.1.12",
```

Comment:
I think the version and revision are a bit mixed on this one. The last version and revision seems to be "v0.1.2".

Reply:
i did not look at these other packages, as i do not maintain them.
i can have a look real quick